docs/developers_guide/index.rst

```diff
@@ -0,0 +1,5 @@
+##########################
+Open edX Developer's Guide
+##########################
+
+This the landing page for the Engineering managed developer's guide. It contains guides relevant to developers of the Open edX platform.  This guide is intended to replace the `(Legacy) Open edX Developer's Guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/>`_.
```

Comment:
Will the community know what we mean by "Engineering managed"?

Reply:
"This the landing page for developer guides created by the edX Engineering team." ?